### PR TITLE
Enable Axe accessibility tests for about page

### PR DESCRIPTION
**Pull request by miguelvaara**
## Reasons for creating this PR
Implemented axe-about.cy.js test to improve accessibility.

## Link to relevant issue(s), if any
#1582.

- Closes #

## Description of the changes in this PR
Fixes the code where it had issues pointed out by the Axe tool on the About page.

Intstructions to the reviewer: Just run the Cypress test like you usually do and focus on checking the accessibility test for the About page.

## Known problems or uncertainties in this PR
-

## Checklist

- [ ] phpUnit tests pass locally with my changes
- [x] I have added tests that show that the new code works, or tests are not relevant for this PR (e.g. only HTML/CSS changes)
- [x] The PR doesn't reduce accessibility of the front-end code (e.g. tab focus, scaling to different resolutions, use of `.sr-only` class, color contrast)
- [x] The PR doesn't introduce unintended code changes (e.g. empty lines or useless reindentation)

### CODE REVIEW

**Comment by osma** [comment]
I noticed that the CI run `cypress-test (push)` had failed, probably for unrelated reasons, but I restarted it just in case to verify that it was just a transient failure.

File: resource/css/skosmos.css [hunk 1]
```diff
@@ -860,6 +860,15 @@ body {
     border-radius: 0;
   }
 
+ /* About page
+   *****************************************/
+  #about-content a {
+    font-size: 0,8ems;
```
Comment (osma) [inline resource/css/skosmos.css:866]: This uses the wrong decimal point (comma) and wrong unit (ems). It should be:

    font-size: 0.8em;

File: src/view/about-info.inc [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {# This is the template for the about page. You can enter any html here and show different language versions by using the lang value in a twig conditional as demonstrated below. #}
 {% block content %}
-  {% if request.lang == 'fi' %}
-    <h4>Tietoja sanastoselaimesta</h4>
+<div id="about-content">
+	{% if request.lang == 'fi' %}
```
Comment (osma) [inline src/view/about-info.inc:4]: SonarCloud complains about tab characters here, please replace with spaces to follow PSR2 conventions.

File: src/view/about-info.inc [hunk 3]
```diff
@@ -12,5 +13,6 @@
     <h4>About</h4>
     <p>Skosmos is a web based open source ontology browser.</p>
     <a href="http://github.com/NatLibFi/Skosmos">Skosmos GitHub repository</a>
-  {% endif %}
+	{% endif %}
```
Comment (osma) [inline src/view/about-info.inc:16]: Same as above, replace tabs with spaces.

**Review by osma — CHANGES_REQUESTED** [review]
Code looks generally good but the commit history is a bit messy. Can you please do the following:

1. Fix errors in skosmos.css (see comment)
2. Convert tabs to spaces in about.inc (see comments)
3. Change the PR title to something more descriptive, e.g. "Enable Axe accessibility tests for about page"
4. When merging, please use the "Squash and merge" option so that all commits are merged into a single commit

File: resource/css/skosmos.css (hunk 1 above)
Comment (miguelvaara) [reply resource/css/skosmos.css]: Fixed

File: src/view/about-info.inc (hunk 2 above)
Comment (miguelvaara) [reply src/view/about-info.inc]: image

**Review by osma — APPROVED** [review]
Very good!

Please merge using "Squash and merge".

**Pull request merged**